import-service/handlers/importFileParser.js

```diff
@@ -14,12 +15,15 @@ export const importFileParser = (event) => {
             Key: objKey
         }).createReadStream();
 
-        console.log("after stream creation");
-
         s3Stream
             .pipe(csv())
             .on('data', (data) => {
-                console.log(data);
+                sqs.sendMessage({
+                    QueueUrl: 'https://sqs.eu-central-1.amazonaws.com/414456242917/catalogItemsQueue',
```

Comment:
you can share such data dynamically between services
you can store in Output section of product-service ARN and URL to you SQS and reference them in import-serivice
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-importvalue.html